src/components/Movies.js

```diff
@@ -11,6 +11,7 @@ const movieCards = movies.map(movie => {
     <Card
       title={movie.title}
       img={movie.poster_path}
+      // alt={movie.title}
```

Comment:
I know we'll do a refactor to remove unused code, just a reminder for us to take this little guy out of here.